boards/arm/b_l072z_lrwan1/doc/index.rst

```diff
@@ -141,6 +141,8 @@ The Zephyr B-L072Z-LRWAN1 Discovery board configuration supports the following h
 +-----------+------------+-------------------------------------+
 | USB       | on-chip    | usb                                 |
 +-----------+------------+-------------------------------------+
+| LoRa      | on-chip    | sx1276                              |
```

Comment:
What does on-chip mean here?

Reply:
Good question. It should perhaps be “in-module” since the Murata module consists of multiple chips under a metal can on a thin PCB substrate (the module should probably be a separate dtsi). For a practical purposes it’s one indivisible unit.

Reply:
Yes, on-board, on-module or in-module would be better.

Reply:
> Yes, on-board, on-module or in-module would be better.

I’ll change it to in-module. Is there a good way to represent system on module chips in Zephyr? Should they be represented as another SoC? From the boards point of view they could just as well be a SoC.